### PR TITLE
[EDU-987] - Add topic on IDs and keys

### PR DESCRIPTION
**Pull request by Sapples**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

Add a page on how to find to Account ID, App ID, and API keys.

* [Quick Wins](https://ably.atlassian.net/browse/EDU-609).
* [Create topic for finding Account ID, App ID, and API key](https://ably.atlassian.net/browse/EDU-987).

## Review

Check the new topic on IDs and Keys:

* [IDs and Keys](https://ably-docs-edu-987-id-to-k6mhn4.herokuapp.com/ids-and-keys/)

### CODE REVIEW

File: content/root/ids-and-keys.textile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  <img src="/images/screenshots/ids-keys/account-id.png" style="width: 70%" alt="Account ID">
+</a>
+
+h2(#app-id). App ID
+
+App IDs are used by operations that affect individual apps, such as creating an API key.
+
+h3(#find-app-id). How to find your app ID
+
+In the "Ably dashboard":https://ably.com/accounts/any select the app you want to find the aspp ID for. Click on the 'Settings' tab:
```
Comment (tbedford) [inline content/root/ids-and-keys.textile:48]: > you want to find the aspp ID for.

Typo.

File: content/root/ids-and-keys.textile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+The "App ID" is displayed under 'Application settings':
+
+<a href="/images/screenshots/ids-keys/app-id.png" target="_blank">
+  <img src="/images/screenshots/ids-keys/app-id.png" style="width: 70%" alt="App ID">
+</a>
+
+h2(#api-key). API key
+
+Ably uses API keys for authentication, either directly ("basic authentication":/core-features/authentication#basic-authentication) or for generating tokens ("token authentication":/core-features/authentication#token-authentication). Each application has at least one API key and each key can have different rights associated with it.
```
Comment (tbedford) [inline content/root/ids-and-keys.textile:62]: > can have different rights associated 

...different *capabilities*?

File: content/root/ids-and-keys.textile [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+API keys contain three parts: the public app ID and public app key ID (which, combined, form the API key ID) and the API key secret. An example API key is: @I2E_JQ.OqUdfg:EVKVTCBlzLBPYJiCZTsIW_pqylJ9WVRB5K9P19Ap1y0@, made up of:
+
+* @I2E_JQ.OqUdfg@ is the public API key ID (made from the public app ID and app key ID)
+** @I2E_JQ@ is the public app ID (the part before the first period)
+** @OqUdfg@ is the public app key ID (after the period and before the colon)
+* @EVKVTCBlzLBPYJiCZTsIW_pqylJ9WVRB5K9P19Ap1y0@ is the API secret key (after the colon)
+
+API keys should be kept secret and only shared with trusted parties.
```
Comment (tbedford) [inline content/root/ids-and-keys.textile:71]: > API keys should be kept secret and only shared with trusted parties.

I would just say "API keys should be kept secret".

File: content/root/ids-and-keys.textile [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+h3(#find-api-key). How to find your API Key
+
+In the "Ably dashboard":https://ably.com/accounts/any select the app you want to find the API keys for. Click on the "API Keys" tab:
+
+<a href="/images/screenshots/ids-keys/api-keys.png" target="_blank">
+  <img src="/images/screenshots/ids-keys/api-keys.png" style="width: 70%" alt="API keys">
+</a>
+
+Your API keys are then individually displayed with only the public key segment showing, click 'show' to reveal your API secret key.
```
Comment (tbedford) [inline content/root/ids-and-keys.textile:81]: For consistency with other paras, we can use colon here, as a diagram follows.

**Review by tbedford — CHANGES_REQUESTED** [review]

File: content/root/ids-and-keys.textile (hunk 1 above)
Comment (Sapples) [reply content/root/ids-and-keys.textile:48]: Good spot!

**Review by tbedford — APPROVED** [review]

**Pull request merged**